build-scripts/graal.js

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/env node
+/*
+ * Copyright 2018 The Closure Compiler Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+/**
+ * @fileoverview
+ *
+ * Build the graal native compiler image for the current OS.
+ * Intended to be run with a working directory of the intended package.
+ */
+
+const {spawn} = require('child_process');
+const fs = require('fs');
+const path = require('path');
+const ncp = require('ncp');
+const {
+  GRAAL_OS,
+  GRAAL_FOLDER,
+  GRAAL_VERSION,
+  GRAAL_URL
+} = require('./graal-env');
+const TEMP_PATH = path.resolve(__dirname, '../temp');
+
+process.on('unhandledRejection', error => {
+  console.error(error);
+  process.exit(1);
+});
+
+// The latest compiler is now compatible with a released version of graal
+const GRAAL_BUILD_FROM_SRC = !Boolean('COMPILER_NIGHTLY' in process.env);
+
+/**
+ * Simple wrapper for the NodeJS spawn command to use promises
+ *
+ * @param {string} command
+ * @param {Object=} spawnOpts
+ * @return {!Promise}
+ */
+function runCommand(command, spawnOpts = {}) {
+  process.stdout.write(`${command}\n`);
```

Comment:
// log command being executed to stdout

at least I think that's what you're doing here.
However, since you're using `process.stdout.write` instead of `console.log`, I'm not certain.

Reply:
Yup - console.log "pretifies" the contents. This is just a straight write.